packages/mui-joy/src/FormLabel/FormLabel.test.tsx

```diff
@@ -35,4 +36,22 @@ describe('Joy <FormLabel />', () => {
     const { container } = render(<FormLabel htmlFor="input" className="foo-bar" />);
     expect(container.firstChild).to.have.attribute('for', 'input');
   });
+
+  it('should inherit htmlFor from FormControl if htmlFor is undefined', () => {
+    const { container } = render(
+      <FormControl>
+        <FormLabel htmlFor={undefined} />
+      </FormControl>,
+    );
+    expect(container.firstChild?.firstChild).to.have.attribute('for');
+  });
+
+  it('should inherit id from FormControl if id is undefined', () => {
+    const { container } = render(
+      <FormControl>
+        <FormLabel id={undefined} />
+      </FormControl>,
+    );
+    expect(container.firstChild?.firstChild).to.have.attribute('id');
+  });
```

Comment:
This test file is only for FormLabel test.

Instead for this fix, the tests should be written in [`packages/mui-joy/src/FormControl/FormControl.test.tsx`](https://github.com/mui/material-ui/blob/master/packages/mui-joy/src/FormControl/FormControl.test.tsx) under the [`Input` section](https://github.com/mui/material-ui/blob/master/packages/mui-joy/src/FormControl/FormControl.test.tsx#L60).

Reply:
Fixed, thank you